values.yaml

```diff
@@ -51,6 +51,7 @@ global:
 
   # Used to enable nats-server
   natsEnabled: true
+  natsReplicas: 3
```

Comment:
@andriisoldatenko what do you think about moving this config to the nats sub package? These global, top-level flags are mostly used to enable/disable the package itself from the umbrella installation.

Reply:
sgtm. but can you pls suggest how to reference to it? i mean how to share it between nats and houston charts?

Reply:
@schnie ☝️

Reply:
Ohh - hmm, missed that part. Should we maybe have a `nats` block, like the istio block up above?